auth4genai/snippets/common/download-manual.jsx

```diff
@@ -0,0 +1,10 @@
+export const DownloadManual = ({
+  folderPath,
+  repository
+}) => {
+  return (
+    <div>
+      Clone <Icon icon="github" iconType="solid" /> <a href={`https://github.com/${repository}.git`} target="_blank" rel="noreferrer" className="link">{repository}</a> and navigate to <code>{folderPath}</code> directory.
+    </div>
+  );
```

Comment:
should be able to reference the DownloadQuickstartButton now inside of here i imagine?
https://github.com/auth0/docs-v2/blob/3188f5fc37ab01d039dc0eadf0a84cf71da3a91e/auth4genai/snippets/download-quickstart/DownloadQuickstartButton.jsx#L3

We have this feature recently added now w/ Download Sample button on these.